docs/tutorials/get-started/blink-an-led.md

```diff
@@ -0,0 +1,416 @@
+
+### Connect your robot to the Viam SDK
+
+The easiest way to get started writing an application with Viam is to navigate to your robot's page on the [Viam app](https://app.viam.com/robots) and select the **Code sample** tab.
+Select your programming language (**Python** or **Golang**), and copy the boilerplate code.
+
+{{% snippet "show-secret.md" %}}
+
+These code snippets import all the necessary libraries and set up a connection with the Viam app in the cloud.
+
+On your local computer, create a file called <file>blink.py</file> or <file>blink.go</file>.
+Paste the boilerplate code from the **Code sample** tab of the Viam app into the file in your code editor, and save the file.
+
+You can now run the code.
+Doing so will ensure that the Viam SDK is properly installed and that the `viam-server` instance on your robot is live.
+
+Run the code by typing the following command into the terminal, with the correct path to your file:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+python3 <INSERT PATH TO YOUR FILE>/blink.py
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+go <INSERT PATH TO YOUR FILE>/blink.go
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+If you successfully configured your robot and it is able to connect to the Viam app, you should see something like this printed to the terminal after running your program.
+What you see here is a list of the various {{< glossary_tooltip term_id="resource" text="resources" >}} that have been configured on your robot in the Viam app.
+
+<img src="../../img/blink-an-led/image4.png" alt="A screenshot from the Visual Studio Code command line that prints the output of print(robot.resource_names) when your Raspberry Pi has correctly connected and initialized with the Viam app. The output is an array of resources that have been pulled from the Viam app. Some of these are the Vision Service, Data Manager, and Board." width="500">
+
+### Write code to blink the LED
+
+In order to interact with the GPIO pins on our Raspberry Pi, you need to import the [board component](/components/board/) from the Viam SDK.
+
+The **Code sample** tab should have automatically added the board import for you, but it doesn't hurt to double-check.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+At the top of your <file>blink.py</file> file, make sure the following line is part of your imports:
+
+```python
+from viam.components.board import Board
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+For this tutorial's Go code, you will also want `"fmt"` (for printing) and `"time"` (for time delays) as well as the board import.
+
+In <file>blink.go</file>, and inside the `import` block at the top of the file, add the following to the import statements without removing any of the other imports:
+
+```go {class="line-numbers linkable-line-numbers"}
+import (
+  "fmt"
+  "time"
+  "go.viam.com/rdk/components/board"
+)
```

Comment:
I said that because I don't want Python users to think they're missing something, or to make Go users wonder why their tab is different from the Python code.

Reply:
How would you go about making the tabs more similar when the Python one is just looking and seeing that the board import is there, while the Go one requires people to add things that do not auto-import? Should I take away these tabs and make an "IMPORTANT" alert that says "Go users also need to import fmt and time. If you are using Go for this tutorial, the top of your code should include these imports:
....
"?

Reply:
No. The tabs being there is good.

python:
At the top of your blink.py file, make sure it includes the following import:

go
At the top of your blink.go file, make sure it includes the following imports:
// code

The `fmt` package enables you to print and the `time` package enables you to add time delays.